docs/creature_text_locale.md

```diff
@@ -8,37 +8,46 @@ This table is used to provide to localized clients with localized string for cre
 
 **Structure**
 
-| Field                                        | Type         | Attributes | Key | Null | Default | Extra | Comment |
-|----------------------------------------------|--------------|------------|-----|------|---------|-------|---------|
-| [entry](#creature_text_locale-entry)         | MEDIUMINT | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [groupid](#creature_text_locale-groupid)     | TINYINT   | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [id](#creature_text_locale-id)               | TINYINT   | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [text\_loc1](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc2](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc3](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc4](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc5](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc6](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc7](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc8](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
+| Field           | Type      | Attributes | Key | Null | Default | Extra | Comment |
+|-----------------|-----------|------------|-----|------|---------|-------|---------|
+| [CreatureID][1] | MEDIUMINT | UNSIGNED   | PRI | NO   | 0       |       |         |
+| [GroupID][2]    | MEDIUMINT | UNSIGNED   | PRI | NO   | 0       |       |         |
+| [ID][3]         | MEDIUMINT | UNSIGNED   | PRI | NO   | 0       |       |         |
+| [Locale][4]     | VARCHAR   |            | PRI | NO   |         |       |         |
```

Comment:
GroupID and ID are TINYINT...

And non integers should still display width. so VARCHAR(4)